src/components/SearchBar/SearchBar.stories.tsx

```diff
@@ -21,22 +34,7 @@ export default {
       },
     },
     label: {
-      control: {
-        options: ['headline', 'claims', 'Test'],
-        type: 'string',
-      },
```

Comment:
Label could be anything in the future, so I think we can just pass any string here.